docs/static/gwadoc.css

```diff
@@ -62,3 +80,13 @@ div.highlight { background-color: #F4F4F4 }
     article table { margin: 2px 0px 2px 0px; }
     article table td { padding: 1px 0px 1px 5px; }
 }
+
+.card {
```

Comment:
Do you anticipate other "card" elements beyond the "relation" one? If not, these styles can be placed on the "relation" class.